src/api/operations/ce.fragments.graphql

```diff
@@ -21,6 +21,15 @@ fragment CeOpportunityFields on CeOpportunity {
   }
 }
 
+fragment ClientCeOpportunitySummaryFields on CeOpportunity {
+  ...CeOpportunitySummaryFields
+  project {
+    id
+    projectName
```

Comment:
The fields that were deprecated are not  removed from `CeOpportunitySummaryFields` above

Reply:
I un-deprecated them, per conversation https://github.com/greenriver/hmis-warehouse/pull/5294#discussion_r2052893847